cocos/2d/CCDrawNode.cpp

```diff
@@ -407,8 +417,16 @@ void DrawNode::onDrawGLLine(const Mat4 &transform, uint32_t flags)
     }
     
     glBindBuffer(GL_ARRAY_BUFFER, 0);
-    
     CC_INCREMENT_GL_DRAWN_BATCHES_AND_VERTICES(1,_bufferCountGLLine);
+
+#if CC_TARGET_PLATFORM == CC_PLATFORM_MAC || CC_TARGET_PLATFORM == CC_PLATFORM_WIN32
```

Comment:
GL_MULTISAMPLE and GL_LINE_SMOOTH only works on desktop system, GLES won't support these features, so it been limited to Windows and MacOS